src/utils/pluralise.js

```diff
@@ -0,0 +1 @@
+export const pluralize = (count, noun, suffix = 's') => `${count} ${noun}${count !== 1 ? suffix : ''}`;
```

Comment:
@AbhishekPAnil This is fine for our needs right now, but creating a plural is not always adding an 's'.  For example, person becomes people.  In the future (not now because we need to launch end February), we can dig deeper and look at https://www.i18next.com/translation-function/plurals